packages/react-components/src/components/loader.tsx

```diff
@@ -8,12 +8,11 @@ const bouncer = keyframes`
     transform: scale(1.75) translateY(-20px);
   }
 `
-const loaderHeight = 37
+const loaderHeight = 52
 const Container = styled.div`
     display: flex;
     align-items: center;
     height: ${loaderHeight}px;
-    padding-top: 15px;
```

Comment:
This padding is moved to the height. Padding is not applied when in a flex layout, causing the loader to overlap the bottom of the gifs when animating